### DISCUSSION TITLE
sendmail doesn't work - Must issue a STARTTLS command first

### DISCUSSION THREAD

**Discussion by Ringoinmel** [leokhoa/laragon#961] [General]
I try to send mail from chamilo and fails.  When I go to Menu > preference > mail Sender > Test mail Sending and it works

However, I got this error in error.log under C:\laragon\bin\sendmail

24/12/30 18:11:38 : Must issue a STARTTLS command first. For more information, go to<EOL> https://support.google.com/a/answer/3221692 and review RFC 3207<EOL>specifications. d9443c01a7336-219dc962d15sm173455385ad.11 - gsmtp<EOL>

I can see several .eml file stills in C:\laragon\bin\sendmail\output folder 

Any ideas or suggestion?  Thanks a lot

**Comment by ppfoong** [comment]
After it shows test OK, did you received an email with the following content in your test email recipient's mailbox? Also check in the spam folder as well.


Subject: Laragon Mail Analyzer: Congratulations! Your configuration is OK

It means you can use mail() function to send email to Internet easily with Laragon. Let's try:

mail('[leokhoa@gmail.com](mailto:leokhoa@gmail.com)', 'Hi leokhoa', 'I like Mail Sender feature.');

Enjoy programming :)

Best wishes,
Laragon.

**Reply by Ringoinmel** [reply]
Yes.  I got the email from Test mail Sending.  But I think the chamilo generated email need to have something called STARTTLS so that it is compatible to use gmail and then it can send out by Laragon

**Reply by ppfoong** [reply]
If it is working, then Laragon is working fine in handling mail() function.

You just need to use the mail() function to send out emails.
